pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -17,6 +17,8 @@ type SubnetSpec struct {
 	IPv4SubnetSize int `json:"ipv4SubnetSize,omitempty"`
 	// Access mode of Subnet, accessible only from within VPC or from outside VPC.
 	// +kubebuilder:validation:Enum=Private;Public;PrivateTGW
+	// +kubebuilder:validation:Required
```

Comment:
This `Required` in label does not matter, right? If we do not set it as required in CRD yaml definition.
To me I do not want to make AccessMode as required, we should have a default value either in definition or in NSX Operator code.

Reply:
If you always want this field to have a value, that means, it's Required. The difference putting  > Required